### PR TITLE
Add introduction (tutorial) to README and ReadTheDocs

### PR DESCRIPTION
**Pull request by vkozareva**
A basic tutorial is now available on the README and ReadTheDocs, using DMD data from the publication. 

Also includes commits related to more user-friendly CLI:
- Improved help text
- Allow subject csv file for `evaluate-sample` command instead of just BAM
- Add three verbosity options to all commands
- Make outputFile required for `create-matrix`

Skeleton of CLI documentation is included, but will be fleshed out in later PR.

### CODE REVIEW

File: docs/introduction.rst [hunk 1]
```diff
@@ -0,0 +1,122 @@
+Introduction
+============
```
[the remark this answers is not in the document]
Comment (vkozareva) [reply docs/introduction.rst]: The other option I was considering was to leave only the installation instructions in the README and keep the introduction on ReadTheDocs. Do you think that might work better?

File: cnv/cli.py [hunk 2]
```diff
@@ -66,48 +71,60 @@ def create_matrix(targetsBedfile, bamfilesFofn, outputFile=None, targetArgfile=N
...
 
     # Read the parameters file.
     targets_params = cPickle.load(open(parametersFile, 'rb'))
     full_targets = targets_params['full_targets']
     targets_to_test = targets_params['parameters'].targets
 
-    # Parse subject bamfile
-    subject_df = CoverageMatrix(unwanted_filters=targets_params['unwanted_filters']).create_coverage_matrix([subjectBamfilePath], full_targets)
+    # Parse subject file
+    if subjectFilePath.endswith('.csv'):
```
[the remark this answers is not in the document]
Comment (vkozareva) [reply cnv/cli.py]: It ignores anyone beyond the first subject in a testing CSV. Once I decided to keep it at single-sample evaluation, this seemed like the simplest way to deal with this, but I realize there are a few different options here:
 - Raise an error if there are multiple subjects in the CSV 
 - Evaluate only the first subject, but make this more explicit in the comments and logging 
    info
 - Allow users to specify which row to evaluate if passing a CSV (seems a bit clunky, but 
    maybe?)

Thoughts?

[the remark this answers is not in the document]
Comment (evolvedmicrobe) [reply cnv/cli.py]: I think an error or logged message seems like the best option.

File: docs/introduction.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (evolvedmicrobe) [reply docs/introduction.rst]: I like em in both places, but thought there might be a way to symbolically link them, etc.

**Pull request merged**